Userland/Games/2048/BoardView.cpp

```diff
@@ -58,7 +58,7 @@ void BoardView::pick_font()
         }
     });
 
```

Comment:
This will cause a desync between the min cell size and the font size used, how about this?
```suggestion
    if (best_font_name.is_empty()) {
        dbgln("Failed to find a good font for size {}, using the default font", m_cell_size / 2);
        auto font = font_database.default_font();
        best_font_name = font.qualified_name();
        best_font_size = font.pixel_size_rounded_up();
    }
````

Reply:
Thanks for the correction. Done!